platform/engineering/design-docs/2020-04-03-separate-content-build.md

```diff
@@ -0,0 +1,486 @@
+# Separate Content Build
+
+**Author(s):** Christopher Valarida  
+**Last Updated:** March 16 2020  
+**Status:** Draft | **In Review** | Approved  
+**Approvers:** Dror Matalon, Andrew Gunsch, Rian Fowler  
```

Comment:
```suggestion
**Approvers:** Dror Matalon, Andrew Gunsch [x], Rian Fowler  
```

I'm OK approving here, I think at a high level most of the major questions are answered and this direction seems solid. Obviously finish getting Rian's approval still, and maybe also CMS team? (see comment below).

Ping Dror and ask if he wants to review this or not; he mentioned to me today he's trying to be less in the weeds on everything now that he's back, so he may opt to pass on this.